client/test/TestFluid.res

```diff
@@ -2852,7 +2852,7 @@ let run = () => {
       aConstructor,
       ~pos=4,
       inputs(list{DeleteWordForward}),
-      "Just ___~",
+      "Just ~___",
```

Comment:
I'm not sure how, but this improvement came for free - the cursor is now at the start of the blank rather than the end.